tests/acceptance/00_basics/04_bundles/dynamic_bundlesequence/dynamic_inputs_based_on_list_variable_dependent_on_class.cf

```diff
@@ -0,0 +1,25 @@
+body common control
+{
+    inputs         => { @(inventory.inputs)  };
+    bundlesequence => { @(inventory.bundles) };
```

Comment:
@kacf this was the missing test we were talking about

@nickanderson let me know if this test doesn't make sense